homeassistant/components/lightwave.py

```diff
@@ -0,0 +1,49 @@
+"""
```

Comment:
Per the above comment about embedded platforms, move this into a folder and rename `__init__.py`

Reply:
When the component was original checked in it was in its own folder. During the last review I was told to put the file here. I don't really want to make this change only to be told to move it again.

A part bumping the version number no other change has been made to this file.

Reply:
Again please rebase your code.  If you look at your PR changes, you're adding entirely new files--not just changing existing ones.